JudoPayDotNet/Models/Validations/PaymentRequiresThreeDSecureTwoModelValidation.cs

```diff
@@ -0,0 +1,19 @@
+using FluentValidation;
+
+namespace JudoPayDotNet.Models.Validations
+{
+	internal class PaymentRequiresThreeDSecureTwoModelValidation : TransactionResultBaseValidation<PaymentRequiresThreeDSecureTwoModel>
```

Comment:
Is this class needed, as PaymentRequiresThreeDSecureTwo is a response object rather than a request?